src/mixins/IndividuMixin.js

```diff
@@ -1,21 +1,45 @@
 import Individu from '@/lib/Individu'
 
+const autoValidatedSteps = [
+    'activite',
+    'nationalite',
+    'handicap',
+    'inapte_travail',
+    'garde_alterne',
+    'handicap',
+    'enfant_a_charge',
+    'taux_incapacite',
+    'ass_precondition_remplie',
+    'aah_restriction_substantielle_durable_acces_emploi',
+    'enfant_place'
+];
+
 export const createIndividuMixin = (props, optional) => {
     return {
         data: function() {
             const id = this.$route.params.id
             const role = id.split('_')[0]
             const { individu } = Individu.get(this.$store.getters.peopleParentsFirst, role, this.$route.params.id, this.$store.state.dates)
+            const autoValidate = autoValidatedSteps.includes(props)
             const value = individu[props]
             return {
                 error: false,
                 individu,
                 id,
                 value,
                 role,
+                autoValidate,
                 optional
             }
         },
+        watch: {
+           value() {
+                if (this.autoValidate) {
```

Comment:
Il faudrait avoir une logique plus importante que `this.autoValidate` par exemple `this.autoValidate && !this.initiallySet` pour s'assurer que la réédition ne fasse pas de passage à l'écran suivant intempestif. (Mais c'est peut-être du zèle de ma part) à confronter aux premiers usages.